conjur/wrapper/http_wrapper.py

```diff
@@ -160,7 +161,7 @@ def __create_ssl_context(ssl_verify: Union[bool, str]) -> Union[bool, ssl.SSLCon
         return False
 
     if ssl_verify is True:
-        ssl_context = ssl.create_default_context()
+        ssl_context = SslContextFactory.create_platform_specific_ssl_context()
     else:
         ssl_context = ssl.create_default_context(cafile=ssl_verify)
```

Comment:
You can move this `if`/`else` into the factory too, so it will be:
```
ssl_context = SslContextFactory.create_ssl_context()
```
Which looks good and makes sense.